<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gui::window</class>
 <widget class="QWidget" name="gui::window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PBus to AMCP</string>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <item>
    <widget class="QGroupBox" name="device_group">
     <property name="title">
      <string>PBus</string>
     </property>
     <layout class="QGridLayout" name="layout_3">
      <item row="0" column="4">
       <widget class="QPushButton" name="serial_open">
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>CasparCG</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="6">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QPushButton" name="caspar_scan">
        <property name="text">
         <string>Scan</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="caspar_port">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="value">
         <number>5250</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Device #</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Host</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4">
       <widget class="QPushButton" name="caspar_connect">
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="serial_port">
        <property name="editable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Serial Port</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QPushButton" name="serial_settings">
        <property name="text">
         <string>Settings...</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="caspar_host"/>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="pbus_device">
        <property name="maximum">
         <number>47</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="registers_group">
     <property name="title">
      <string>Registers</string>
     </property>
     <layout class="QVBoxLayout" name="layout_4">
      <item>
       <widget class="QScrollArea" name="registers_scroll">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="registers">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>572</width>
           <height>581</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="layout_2"/>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>serial_port</tabstop>
  <tabstop>pbus_device</tabstop>
  <tabstop>serial_open</tabstop>
  <tabstop>serial_settings</tabstop>
  <tabstop>caspar_host</tabstop>
  <tabstop>caspar_port</tabstop>
  <tabstop>caspar_connect</tabstop>
  <tabstop>caspar_scan</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
